plugins/foreman_templates/4.0/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: plugin
+title: Foreman Templates 4.0 Manual
+version: 4.0
+---
+
+# 1. {{ page.title }}
+
+This plugin enables synchronization of provisioning templates, partition tables, job templates from external git repository and/or file system. It can be used both with the [community templates](https://github.com/theforeman/community-templates) repository or with any other source of templates that has the same directory structure. Please note that Foreman core provides tracking of changes, browsing the history and reverting natively, but if you need more advanced tools such as branching, code reviews etc. you might be interested in this plugin.
+
+You can use the native git support for simple syncrhonization with remote git repositories, which is usually useful for syncing new template versions from community repository or for exporting local changes to contribute back. Or if you have more complicated workflow, you can point the plugin to work with local directory. This way, you can use any VCS (including git) to control the versioning of that directory and use this plugin to sync whenever the directory is at desired version.
+
+# 2. Installation
+
+You can install the plugin using the foreman-installer (recommended way). To do that, run the installer with following argument
+
+    foreman-installer --enable-foreman-plugin-templates
+
+If installed foreman from git, you can simply add foreman-templates gem as one of the dependencies. Don't forget to run migrations and seed afterwards.
+      
+# 3. Configuration
+
+After the installation there are new settings available. These are the default setting for both importing and exporting tasks. They can be overridden on each synchronization run and serve only as default so users don't have to specify the same value on every run. You will find them under Administer -> Settings -> TemplateSync. Only administrator can modify their values. Following table explains the attribute behavior. Note that some are only used on import or export actions.
+
+<table class="table table-bordered table-condensed table-striped">
```

Comment:
I see you're going for alphabetical setting order - but many options are rarely used, does it makes sense to put common options first?